helix-term/src/ui/editor.rs

```diff
@@ -541,6 +554,43 @@ impl EditorView {
         }
     }
 
+    /// Render bufferline at the top
+    pub fn render_tabs(
+        editor: &Editor,
+        view: &View,
+        viewport: Rect,
+        surface: &mut Surface,
+        theme: &Theme,
+    ) {
+        let pb = PathBuf::from(" [scratch] "); // default filename to use for scratch buffer
+        surface.clear_with(viewport, theme.get("ui.statusline"));
+        let mut len = 0usize;
+        editor.documents().for_each(|doc| {
+            let fname = doc
+                .path()
+                .unwrap_or_else(|| &pb)
+                .file_name()
+                .unwrap_or_default()
+                .to_str()
+                .unwrap();
+
+            let style = if view.doc == doc.id() {
+                theme.get("ui.background")
+            } else {
+                theme.get("ui.statusline")
+            };
```

Comment:
Introducing new theme scopes like `ui.bufferline` and `ui.bufferline.active` which fall back to these values would be a nice addition.

Reply:
Is there a straighforward way to have "ui.bufferline" fallback to "ui.statusline"? Reading the docs it seems that it would simply fallback to "ui".

Reply:
You could do something like `theme.try_get("ui.bufferline").unwrap_or_else(|| theme.get("ui.statusline")`. That would behave differently from how other theme fallbacks work though, so I think we should just introduce a new `ui.bufferline` and copy `ui.statusline` values in the existing themes